hugegraph-api/src/main/java/com/baidu/hugegraph/auth/HugeAuthenticator.java

```diff
@@ -81,7 +81,8 @@ default void setup(final Map<String, Object> config) {
 
     @Override
     default User authenticate(final Map<String, String> credentials)
-                                     throws AuthenticationException {
+            throws AuthenticationException {
```

Comment:
align with final

Reply:
I don't think so we could algin with "final"
if we break line from "credentials",we can align with "final",because "final Map<String, String> credentials" are the entire content in parenthesis
but "throws" is out of parenthesis

Reply:
#1996